.github/workflows/update_version.yml

```diff
@@ -1,15 +1,26 @@
 name: Update Version on Docker Files
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  POSTGRES_VERSION: ${{ github.event.inputs.postgres_version }}
 on:
- workflow_dispatch:
+  workflow_dispatch:
     inputs:
       project_version:
-        description: 'Version number to be released e.g 10.0.3'
+        description: "Version number to be released e.g 10.0.3"
       postgres_version:
         required: false
-        description: 'Postgres version number. If it is empty, postgres version will not be changed'
+        description: "Postgres version number. If it is empty, postgres version will not be changed"
```

Comment:
We have 2 supported PG versions now. 12 and 13. Which one should be supplied here? Only 13 minor version?

What about PG12?

Reply:
only 13. 12 is additional one that we added. I have an issue to update all versions using postgres compatibility matrix like structure at 2 item in the list below
https://microsoft.sharepoint.com/:x:/r/teams/azurepostgres-product-all/_layouts/15/Doc.aspx?sourcedoc=%7B631A45D8-F917-4187-BCE1-81A6E919FC45%7D&file=Devops%20Tasks.xlsx&action=default&mobileredirect=true&share=IQHYRRpjF_mHQbzhgabpGfxFAagZkpXW3g3IXa9aiXEq5VY&wdhostclicktime=1629746810096&cid=393591b4-85b0-40ee-9262-3fb389c738ba